data/engines.txt

```diff
@@ -419,7 +419,7 @@ outfit "A860 Atomic Thruster"
 	"flare sprite" "effect/atomic flare/huge"
 		"frame rate" 10
 	"flare sound" "atomic huge"
-	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. But, you may need to upgrade your ship's power and cooling systems to handle it."
+	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. However, you may need to upgrade your ship's power and cooling systems to handle it."
```

Comment:
No, that is still underwhelming.

This is literally the most powerful engine in human space; we need a description that highlights its awesome power.
Something relating to the huge power drain could work, but as it is, this doesn't have much impact.